### PR TITLE
Feat : Added Navigation ( Desktop )

### PR DESCRIPTION
**Pull request by srejitk**
The following merge request adds the following:

1. Left Aligned Navigation
2. Spaced Navigation
3. The Documentation for the same

Left Navbar
spaced navbar

### CODE REVIEW

File: docs/Navigation.html [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Navigation</title>
+    <link rel="stylesheet" href="/main.css">
+    <link rel="stylesheet" href="/stylesheets/components/Navigation/Navigation.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/styles/night-owl.min.css"
+        integrity="sha512-i5X6Fdn/ZqvGSqPrdMa3FgcpXM/Nr6YccSKFYT93zljl/HZDEpvBbE5Pxp91eiWGccZLrL/LDQJd7fjTRYsVaA=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
```
Comment (nishasen) [inline docs/Navigation.html:16]: Try adding links in separate css file

File: docs/Navigation.html [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+</head>
+
+<body>
+    <div class="main-container">
+        <!-- Header HTML -->
+        <header id="header" class="iris__header">
+            <nav class="iris__navbar">
+                <div class="iris__navbar__logo__section">
+                    <a class="home__section">
```
Comment (nishasen) [inline docs/Navigation.html:25]: Add href attribute

**Review by nishasen — APPROVED** [review]
Code looks clean and good, easily understandable.

File: stylesheets/components/Navigation/Navigation.css [hunk 3]
```diff
@@ -0,0 +1,67 @@
+
+    /* Navbar  */
+    .navbar {
+        display: flex;
+        justify-content: flex-start;
+        align-items: center;
```
Comment (shuntzhere) [inline stylesheets/components/Navigation/Navigation.css:6]: You can use utilities for these.

**Review by shuntzhere — APPROVED** [review]
Great use of comments throughout the codes.  No changes are to be done.

File: docs/Navigation.html (hunk 1 above)
Comment (srejitk) [reply docs/Navigation.html:16]: Yes , will do. Thank You.

File: docs/Navigation.html (hunk 2 above)
Comment (srejitk) [reply docs/Navigation.html:25]: Thank You. Will do.

File: stylesheets/components/Navigation/Navigation.css (hunk 3 above)
Comment (srejitk) [reply stylesheets/components/Navigation/Navigation.css:6]: Yes, understood.

**Pull request merged**